app/views/lessons/show.jbuilder

```diff
@@ -1,6 +1,6 @@
 json.extract! @lesson, :id, :title, :description, :video_link, :author_id, :category_id, :status, :created_at,
               :image_link, :rating, :votes_count
-json.views_count @views_count
+json.views_count @lesson.views_count
```

Comment:
We do not really need an additional field views_count on the lessons table. We always can calculate if using @lesson.lesson_views.size. This can have sense  only in case, if we will have a really huge number of records in lesson_views table and calculation will took a lot of resources.